BackEndFlask/controller/Routes/Assessment_task_routes.py

```diff
@@ -27,6 +30,22 @@
 @badTokenCheck()
 @AuthCheck()
 def get_all_assessment_tasks():
+    if(request.args and request.args.get("course_id")):
+        course_id = int(request.args.get("course_id"))
+        course = get_course(course_id)
```

Comment:
No need to call get_course here since we don't use any information from the course except the course_id, which we already have.

Reply:
Fixed!